modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -172,6 +172,16 @@ image::dashboard-built-in-infrastructure.png[Example: Infrastructure Dashboard]
 | *Custom metrics* a|
 
 The *Custom metrics* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. See xref:anypoint-custom-metrics-connector.adoc[Anypoint Custom Metrics Connector].
+
+| *Application Network* a|
+
+The *Application Network* tab displays only if you have a Titanium subscription and select an app running on a supported version of Mule. 
+
+The chart on this tab shows the xref:visualizer::troubleshooting-visualization.adoc[Troubleshooting visualization] for the selected app. Additionally, it shows only direct callers and callees for the selected app over the most recent seven days. 
```

Comment:
@sarathecoubian can we add a note here that the app must be started. 
The network is only visible for *started* apps and the connections are based on the time picker.

Reply:
```suggestion
The chart on this tab shows the xref:visualizer::troubleshooting-visualization.adoc[Troubleshooting visualization] for the selected app. Additionally, it shows only direct callers and callees for the selected app over the most recent seven days. The apps must be deployed, and the connections are based on the selected time range.
```